src/parser/components.rs

```diff
@@ -131,15 +131,27 @@ impl From<Component<'_>> for Other {
 
 impl From<Component<'_>> for InnerComponent {
     fn from(component: Component) -> Self {
-        Self {
-            properties: component
-                .properties
+        fn is_multi(property: &Property) -> bool {
+            property.name.to_string() == "ATTENDEE"
+        }
+
+        let (multi, single): (Vec<_>, Vec<_>) =
+            component.properties.into_iter().partition(is_multi);
```

Comment:
there is probably a way to create an `Iterator` over all the multi properties and one over all the non-multi properties and iterate over those without copying into two new `Vec`s